fake-store/build.gradle

```diff
@@ -40,8 +40,8 @@ android {
     }
 
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
```

Comment:
https://developer.android.com/build/releases/gradle-plugin#jdk-17-agp

Reply:
I may be wrong but isn't just say that JDK 17 is now required to run Gradle?
You can run Gradle under the installed JDK 17 and compile for older version of Java or I'm wrong?

Reply:
I'm not completely sure but I have noticed warnings (and in other projects errors) when using different java versions for different purposes

Reply:
I cannot try currently but are you sure that with this change it will still work under Android 2.3?

Reply:
I am not sure of that